scripts/leetcode.js

```diff
@@ -600,22 +606,40 @@ function startUploadCountDown() {
     }
   }, 10000);
 }
-/* start upload will inject a spinner on left side to the "Run Code" button */
-function startUpload() {
-  elem = document.getElementById('leethub_progress_anchor_element')
-  if (elem !== undefined) {
-    elem = document.createElement('span')
-    elem.id = "leethub_progress_anchor_element"
+
+/* we will need specific anchor element that is specific to the page you are in Eg. Explore */
+function insertToAnchorElement(elem){
+  if(document.URL.startsWith("https://leetcode.com/explore/")) {
+    // means we are in explore page
+    target = document.getElementsByClassName('action')[0].getElementsByClassName('row')[0].getElementsByClassName('col-sm-6')[1]
```

Comment:
assumption ur making here is all these fields are available. u need to do null check for this, unless this is guaranteed to be present within leetcode. plz verify the ubiquity of this structure @sbibek